jet_images_ml.py

```diff
@@ -281,14 +281,14 @@ def optimize(num_iterations, starting_iteration=0):
 #    print y_valid.shape
 
 
-    #features_class=np.concatenate(classifier,axis=0)
+    features_class=np.concatenate(classifier,axis=0)
     
     #feature_labels=np.concatenate(labels,axis=0)
     
-    #features_class=np.concatenate((features_class,feature_labels),axis=1)
+    features_class=np.concatenate(features_class,axis=1)
```

Comment:
The array feature_class is 3 dimensional. I couldn't save it as a .csv file. Also we don't need a feature column for this file right?

Reply:
you do not need to save as a csv if it isn't convenient. there are many format to choose from e.g. json, npy/npz, pickle etc. When you are struggling with things like this google + stack overflow are your friend.